mod/phpmailer/start.php

```diff
@@ -226,37 +209,55 @@ function phpmailer_send($to, $to_name, $subject, $body, array $bcc = NULL, $html
 
 	$is_ssl    = elgg_get_plugin_setting('ep_phpmailer_ssl', 'phpmailer');
 	$ssl_port  = elgg_get_plugin_setting('ep_phpmailer_port', 'phpmailer');
-	try{
-	if ($is_smtp && isset($smtp_host)) {
-		//$phpmailer->IsSMTP();
-		//$phpmailer->Host = $smtp_host;
-		//$phpmailer->SMTPAuth = false;
-		if ($smtp_auth) {
-			//$phpmailer->SMTPAuth = true;
-			//$phpmailer->Username = elgg_get_plugin_setting('phpmailer_username', 'phpmailer');
-			//$phpmailer->Password = elgg_get_plugin_setting('phpmailer_password', 'phpmailer');
-
-			if ($is_ssl) {
+
+	try {
+
+		if ($is_smtp && isset($smtp_host)) {
+
+			if ($smtp_auth && $is_ssl) {
+
 				$phpmailer->SMTPSecure = "tls";
 				$phpmailer->Port = $ssl_port;
+				
 			}
+		} else {
+			// use php's mail
+			$phpmailer->IsMail();
 		}
-	}
-	else {
-		// use php's mail
-		$phpmailer->IsMail();
-	}
-	//error_log("Sending email from => " . $phpmailer->FromName );
-	$return = $phpmailer->Send();
-	//error_log("Sent email from => " . $phpmailer->FromName );
-	} catch(phpmailerException $e) {
-		error_log($e->errorMessage());
+	
+		$return = $phpmailer->Send();
+
+	} catch (phpmailerException $e) {
+		//error_log($e->errorMessage());
```

Comment:
Delete?

Reply:
@Phanoix can i remove these error logs? (i'm not sure who put these in or whether or not it's a good idea to remove debugging code entirely)

Reply:
This one seems to be one of ours, you can probably get rid of it now.